app/src/androidMain/kotlin/App.kt

```diff
@@ -0,0 +1,13 @@
+package com.juul.sensortag
+
+import android.app.Application
+import com.juul.tuulbox.logging.ConsoleLogger
+import com.juul.tuulbox.logging.Log
+
+class App : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        Log.dispatcher.install(ConsoleLogger)
```

Comment:
I'm now realizing I don't provide a Logcat logger in Tuulbox and this feels like a gap to me.

Reply:
Agreed. Gonna be fun having Gradle properly handle the existence of a JVM **and** Android artifact. 😝

Reply:
I _believe_ Coroutines did it by making the Android artifact separate from the "bundle" of multiplatform artifacts.

![Screen Shot 2021-07-07 at 12 33 24 PM](https://user-images.githubusercontent.com/98017/124818270-ac24b880-df1f-11eb-9f89-2aa90f815e03.png)

Reply:
https://github.com/JuulLabs/tuulbox/issues/74